### ISSUE TITLE
DOI compatibility

### ISSUE THREAD

**Issue by zagorGit** [seek4science/seek#107]
Minted DOIs for data files/documents, assays/studies/investigations not recognized by datacite.org nor crossref.org nor identifiers.org....
Any possibility to make them "detectable"?

**Comment by alaninmcr** [comment]
Do you have an example of an object where the DOI is not recognized?

Also which instance of SEEK is it on?

**Comment by zagorGit** [comment]
DOI minted under project  161 at fairdomhub.org
e.g.
10.15490/fairdomhub.1.datafile.3116.1
10.15490/fairdomhub.1.datafile.3119.1
http://doi.org/10.15490/FAIRDOMHUB.1.STUDY.670.1

**Comment by zagorGit** [comment]
Update:

FAIRDOMHUb is listed by FAIRsharing and re3data.org. However it is not listed by Springer Nature and it cannot mint ''DataCite DOIs'' (DOI compatible with DataCite Metadata Schema 4.3). 
It is stated that ''Repository managers to investigate listing repositories with re3data.org in the first instance and, for those that meet the criteria, applying for listing with Scientific Data'' [https://www.springernature.com/gp/authors/research-data-policy/faqs/12327154]. Therefore, it would be appreciated if FAIRDOMHub/seek repository managers could applying for listing with Scientific Data so that FAIRDOMHub minted DOIs' become recognizable by DOI Registration Agencies.

Thank you and kind regards

**Comment by fbacall** [comment]
What do you mean by "recognized" exactly?

We are using DataCite to provide our DOIs, so they should recognize them at least.

```
$ curl -LH "Accept: application/vnd.datacite.datacite+xml" https://doi.org/10.15490/fairdomhub.1.datafile.3116.1
<?xml version="1.0" encoding="UTF-8"?>
<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd">
  <identifier identifierType="DOI">10.15490/FAIRDOMHUB.1.DATAFILE.3116.1</identifier>
  <creators>
    <creator>
      <creatorName nameType="Personal">Zagorscak, Maja</creatorName>
      <givenName>Maja</givenName>
      <familyName>Zagorscak</familyName>
    </creator>
  </creators>
  <titles>
    <title xml:lang="en-gb">/reports/Merged.aa.html</title>
  </titles>
  <publisher>FAIRDOMHub</publisher>
  <publicationYear>2019</publicationYear>
  <resourceType resourceTypeGeneral="Other">CreativeWork</resourceType>
  <dates>
    <date dateType="Issued">2019</date>
  </dates>
  <version/>
  <descriptions>
    <description xml:lang="en-gb" descriptionType="Abstract">Supplementary HTML 1 Multiple sequence alignments using MAFFT v7.271 and MView v1.65. Paralogue cluster on representative and alternative sequences, at least one from each of the four genotypes. Clusters containing 8-16 sequences.</description>
  </descriptions>
</resource>
```

There is a bug, however with the `resourceType` and `resourceTypeGeneral` not being set properly, so they appear as "Other". Maybe this is the problem?

**Comment by fbacall** [comment]
Did not intend to close...

**Comment by zagorGit** [comment]
Update with e.g. on https://fairdomhub.org/data_files/3116?version=1

It seems one can query [](https://dx.doi.org/) with `10.5281/zenodo.3552846` (DOI 'badge' on [zenodo.org](url) or [github.com](url) page) or `10.15490/fairdomhub.1.datafile.3116.1` (shown as DOI value/item at [fairdomhub.org](url)) or `http://doi.org/10.15490/FAIRDOMHUB.1.DATAFILE.3116.1` (shown as Citation at [fairdomhub.org](url)); however [](https://search.datacite.org/) only recognizes following format:
`doi:10.5281/ZENODO.3552846`
or
`doi:10.15490/FAIRDOMHUB.1.DATAFILE.3091.1`

- Maybe information provided at [fairdomhub.org](url) regarding doi could be reformatted to prevent potential future obstacles regarding recognition?
- It would till be useful to list the FAIRDOMHub as repository with Springer Nature to facilitate data hosting at it and reporting it in SN journals

**Comment by mfenner** [comment]
@zagorGit https://search.datacite.org/ui is an old, deprecated search interface that will be retired by the end of the year.

dx.doi.org has been deprecated in favor of doi.org.

DataCite content negotiation is documented at https://support.datacite.org/docs/datacite-content-resolver.

Searching for specific DOIs in DataCite Search should be improved and is something DataCite will be working on.

**Comment by zagorGit** [comment]
Thank you for the information.
As i understand, `https://citation.crosscite.org/ `followed by  `curl -LH "Accept: application/vnd.citationstyles.csl+json, application/rdf+xml" http://doi.org/...` is the way to prove how FAIRDOMHub DOIs are registered at DataCite, for now.

**Comment by mfenner** [comment]
Not really. https://citation.crosscite.org/ is for formatting citations. Please don't use content negotiation to check the existence of a DOI. If you want to know whether a DOI is registered with DataCite, you can use 
https://doi.org/ra/10.15490/fairdomhub.1.datafile.3116.1. This API call will return either 

```
[
  {
    "DOI": "10.15490/fairdomhub.1.datafile.3116.1",
    "RA": "DataCite"
  }
]
```

or 

```
[
  {
    "DOI": "10.15490/fairdomhub.1.datafile.XXX",
    "status": "DOI does not exist"
  }
]
```

**Comment by zagorGit** [comment]
The original issue started with:
 _...We also cannot confirm that the DOIs have been registered with DataCite and recommend that you either query this with DataCite .... Although the DOI resolve, they do not appear in DataCite...._
I'm not sure this will be enough.

**Comment by alaninmcr** [comment]
Michael O'Donohue has raise the issue of DOIs and mapping to the DataCite schema (now at version 4.4). It would be a good thing if it was checked.,
